api/ApiHandler.py

```diff
@@ -1,21 +1,34 @@
-from flask_restful import Api, Resource, request
-
-class ApiHandler(Resource):
-  def get(self):
-    search_term = request.args.get("query")
-    ## TODO: return err if query is invalid twitter handle
-    ## TODO: use search_term to return proper array of words
-    return {
-      'resultStatus': 'SUCCESS',
-      'words': [
-                {
-                    "text": 'told',
-                    "value": 64
-                },
-                {
-                    "text": "hi", 
-                    "value": 1
-                }
-            ]
-      }
+from flask_restful import Resource
+import requests;
+import os;
 
+from flask import Flask, jsonify, json
+
+
+class ApiHandelr(Resource):
+  url = 'https://api.twitter.com/1.1/statuses/user_timeline.json'
+  Twitter_Barear_Token= os.environ.get('Twitter_Barear_Token')
+
+  header = {"Authorization":  f"Bearer {Twitter_Barear_Token}"}
+  def getTweets(self):
+      userName = requests.args.get("userName")
+      searchWord=  requests.args.get("word")
+      params = {'screen_name': userName};
+      response = requests.get(url, auth= header, params = params)
+
+      if response.status_code != 200 :
+          raise Exception(
+              "Request returned an error: {} {}".format(
+              response.status_code, response.text    
+              )
+          )
+
+      print('tweets');
+      tweets = response.json();
+      tweet_obj = json.loads(tweets)
+      tweets_filter = [x for x in tweet_obj if x['text'].find(searchWord) != -1]
```

Comment:
I didn't notice this before, but why are we using searchWord? What is that? I don't think the client needs to send that to us at all!